htdocs/comm/action/index.php

```diff
@@ -968,6 +973,14 @@
 			//print 'Event '.$i.' id='.$event->id.' (start='.dol_print_date($event->datep).'-end='.dol_print_date($event->datef);
 			//print ' startincalendar='.dol_print_date($event->date_start_in_calendar).'-endincalendar='.dol_print_date($event->date_end_in_calendar).') was added in '.$j.' different index key of array<br>';
 		}
+
+		$parameters['obj'] = $obj;
+		$reshook = $hookmanager->executeHooks('eventElements', $parameters, $event, $action); // Note that $action and $object may have been modified by some hooks
```

Comment:
For new hook, we should use a method name that start with 
"hook..."
so
hookEventElements